gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -145,6 +145,22 @@ def run(
         return wti_list, ta_metrics
 
 
+def _sample_random_directions(num_samples: int) -> List[Unit3]:
+    """Samples num_samples Unit3 3D directions.
+    The sampling is done in 2D spherical coordinates (azimuth, zenith) values, and then converted to Euclidean
+    coordinates.
+
```

Comment:
@akshay-krishnan could you mention why it's preferred to do the sampling in a different coordinate system? theoretical justification etc